src/v/redpanda/admin_server.cc

```diff
@@ -2793,6 +2797,103 @@ void admin_server::register_cluster_routes() {
 
           co_return ss::json::json_return_type(ret);
       });
+
+    register_route<publik>(
+      ss::httpd::cluster_json::get_partition_balancer_status,
+      [this](std::unique_ptr<ss::httpd::request> req)
+        -> ss::future<ss::json::json_return_type> {
+          vlog(logger.debug, "Requested partition balancer status");
+
+          using result_t = std::variant<
+            cluster::partition_balancer_overview_reply,
+            model::node_id,
+            cluster::errc>;
+
+          result_t result
+            = co_await _controller->get_partition_balancer().invoke_on(
```

Comment:
nit: usually we encapsulate request forwarding logic in `..._frontend` components.

Reply:
I thought about it, felt wasteful creating a whole service just for one function.